resource_obmcs_core_console_history.go

```diff
@@ -110,6 +110,11 @@ func (s *ConsoleHistoryResourceCrud) Get() (e error) {
 	return
 }
 
+func (s *ConsoleHistoryResourceCrud) Delete() (e error) {
+	e = nil
```

Comment:
I'm probably missing some context. It looks like this would silently succeed without doing the delete?